### ISSUE TITLE
An error PharException occurred when trying to update files maxmind.

### ISSUE THREAD

**Issue by vdbin** [stevebauman/location#177]
Hello. In version 7.3.0 an error occurs when trying to update maxmind. The error also occurs on a fresh install laravel.

```
sail php artisan location:update                                                         
Updating driver [Stevebauman\Location\Drivers\MaxMind]...

   PharException 

  phar error: attempted to extract non-existent file or directory "GeoLite2-City.mmdb" from phar "/var/www/html/database/maxmind/maxmind.tar.gz"

  at vendor/stevebauman/location/src/Drivers/MaxMind.php:48
     44▕         $file = $this->discoverDatabaseFile(
     45▕             $archive = new PharData($tarFilePath)
     46▕         );
     47▕ 
  ➜  48▕         $archive->extractTo($storage->path('/'), $file->getFilename(), true);
     49▕ 
     50▕         file_put_contents(
     51▕             $this->getDatabasePath(),
     52▕             fopen($storage->path($file->getFilename()), 'r')

      +15 vendor frames 

  16  artisan:13
      Illuminate\Foundation\Application::handleCommand()
```

Thank you.

**Comment by stevebauman** [comment]
Fixed in [v7.3.1](https://github.com/stevebauman/location/releases/tag/v7.3.1). Apologies for the inconvenience. Run `composer update` and you should be all set! 🙏

**Comment by 3likayed** [comment]
> Fixed in [v7.3.1](https://github.com/stevebauman/location/releases/tag/v7.3.1). Apologies for the inconvenience. Run `composer update` and you should be all set! 🙏

faced the same issue also after updating to the last version
